crypto/cross_sign_ssss.go

```diff
@@ -100,6 +100,12 @@ func (mach *OlmMachine) GenerateAndUploadCrossSigningKeys(ctx context.Context, u
 		return "", nil, fmt.Errorf("failed to publish cross-signing keys: %w", err)
 	}
 
+	// Trust the master key
+	err = mach.SignOwnMasterKey(ctx)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to sign own master key: %w", err)
+	}
+
```

Comment:
@tulir I think this is the only part of this PR that I'm not quite sure if it's the right thing to do. I think it is, but I'm not 100% sure.

Reply:
(Discussed this out of band and we agreed that this is correct.)